### ISSUE TITLE
[openpanel] makes triggering msgboxes to visually hang

### ISSUE THREAD

**Issue by umlaeute** [pure-data/pure-data#296]
clicking a `[bang(` connected to an `[openpanel]` will open the file-dialog (as expected). while the openpanel is open, the `[bang(` msgbox will have it's regular (non-highlighted) outline.
once the file dialog gets closed (e.g. by selecting a file), the msgbox gets activated (the borders get highlighted) - but it never goes back to "normal": the msgbox just hangs there "activated", until the window is redrawn.

![bang-openpanel](https://user-images.githubusercontent.com/214034/35332371-0b070606-010b-11e8-99d6-e3e3e335f12c.png)

this is a minor issue, but has raised questions in the last few years...probably time to fix it.

**Comment by umlaeute** [comment]
i'm currently seeing this on `tcl/tk 8.6` running on Debian/sid[amd64].

running with `-d 1`, i get:
~~~
pdtk_openpanel {d56398d27b840} {}
.x56398d278190.c itemconfigure .x56398d278190.t56398d278650R -width 5
.x56398d278190.c itemconfigure .x56398d278190.t56398d278650R -width 1
~~~

which indicates that this is an issue with tcl/tk

**Comment by umlaeute** [comment]
fixed with 54de8033c
